blueocean-dashboard/src/main/js/components/ActivityDetailsRow.jsx

```diff
@@ -87,7 +87,7 @@ class ActivityDetailsRow extends Component {
                 <TableCell linkTo={runDetailsUrl}>{run.id}</TableCell>
                 <TableCell linkTo={runDetailsUrl}><CommitId commitId={run.commitId} /></TableCell>
                 { isMultibranch && <TableCell linkTo={runDetailsUrl}>{branchName}</TableCell> }
-                <TableCell linkTo={runDetailsUrl}><RunMessageCell run={run} t={t} /></TableCell>
+                <TableCell><RunMessageCell linkTo={runDetailsUrl} run={run} t={t} /></TableCell>
```

Comment:
Why did you change where the linkTo happens? from what I remeber one of the reasons we need to have the link in the TableCell is to properly cover the whole cell. This may not be an issue here, just wondering

Reply:
I had to remove the whole link over the cell because a tags cannot contain other a tags, so to have links in the message, I had to remove the a tag over the whole cell and only add it to the text that is not linked to any jira issue